utils/results/conversion/summaryparser/summaryparser.go

```diff
@@ -69,25 +70,42 @@ func (sc *CmdResultsSummaryConverter) validateBeforeParse() (err error) {
 	return
 }
 
-func (sc *CmdResultsSummaryConverter) ParseViolations(target results.ScanTarget, scaResponse services.ScanResponse, applicabilityRuns ...*sarif.Run) (err error) {
+func (sc *CmdResultsSummaryConverter) ParseScaIssues(target results.ScanTarget, violations bool, scaResponse results.ScanResult[services.ScanResponse], applicableScan ...results.ScanResult[[]*sarif.Run]) (err error) {
+	if violations {
+		return sc.parseScaViolations(target, scaResponse, applicableScan...)
```

Comment:
same as before - can we have both vuln and violations?

Reply:
see above, its the same logic for the other methods as well (the Jas Parsing methods)